### PR TITLE
Feat/view details member

### PR DESCRIPTION
**Pull request by sietsedrent**

### CODE REVIEW

File: Kbs.Wpf/User/ViewUser/ViewUserDetail/ViewUserValuesDetailedReservationViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        {
+            get => _startTime;
+            set
+            {
+                SetField(ref _startTime, value);
+                OnPropertyChanged(nameof(StartTimeString));
+            }
+        }
+
+        public string StartTimeString => StartTime.ToString("dd-MM");
```
Comment (SilGosker) [inline Kbs.Wpf/User/ViewUser/ViewUserDetail/ViewUserValuesDetailedReservationViewModel.cs:36]: Jaar mist

File: Kbs.Wpf/User/ViewUser/ViewUserIndex/ViewUserValuesIndexViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                if (SetField(ref _selectedRole, value))
+                {
+                    OnRoleChanged();
+                }
+            }
+        }
+
+        private void OnRoleChanged()
+        {
+        }
```
Comment (SilGosker) [inline Kbs.Wpf/User/ViewUser/ViewUserIndex/ViewUserValuesIndexViewModel.cs:36]: OnRoleChanged lijkt redundant

**Review by SilGosker — CHANGES_REQUESTED** [review]

File: Kbs.Wpf/User/ViewUser/ReadUserDetail/ReadUserDetailedPage.xaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+﻿<Page x:Class="Kbs.Wpf.User.ViewUser.ViewUserDetail.ViewUserValuesDetailedPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+      xmlns:local="clr-namespace:Kbs.Wpf.User.ViewUser.ViewUserDetail"
+      mc:Ignorable="d" 
+      d:DesignHeight="450" d:DesignWidth="800"
+      Title="ViewUserValuesDetailedPage">
```
Comment (turtleman31) [inline Kbs.Wpf/User/ViewUser/ReadUserDetail/ReadUserDetailedPage.xaml:9]: change to read

File: Kbs.Wpf/User/ViewUser/ReadUserDetail/ReadUserDetailedReservationViewModel.cs [hunk 4]
```diff
@@ -0,0 +1,53 @@
+﻿using Kbs.Business.Helpers;
+using Kbs.Business.Reservation;
+using Kbs.Wpf.Components;
+
+namespace Kbs.Wpf.User.ViewUser.ViewUserDetail
```
Comment (turtleman31) [inline Kbs.Wpf/User/ViewUser/ReadUserDetail/ReadUserDetailedReservationViewModel.cs:5]: change to read

File: Kbs.Wpf/User/ViewUser/ReadUserDetail/ReadUserDetailedPage.xaml.cs [hunk 5]
```diff
@@ -0,0 +1,65 @@
+﻿using Kbs.Business.Reservation;
+using Kbs.Business.User;
+using Kbs.Data.Reservation;
+using Kbs.Data.User;
+using Kbs.Wpf.BoatType.Read.Details;
+using Kbs.Wpf.Reservation.Read.Details;
+using System.Windows.Controls;
+using System.Windows.Input;
+
+namespace Kbs.Wpf.User.ViewUser.ViewUserDetail
```
Comment (turtleman31) [inline Kbs.Wpf/User/ViewUser/ReadUserDetail/ReadUserDetailedPage.xaml.cs:10]: change to read

File: Kbs.Wpf/User/ViewUser/ReadUserDetail/ReadUserDetailedViewModel.cs [hunk 6]
```diff
@@ -0,0 +1,52 @@
+﻿using Kbs.Business.Helpers;
+using Kbs.Business.User;
+using Kbs.Wpf.Components;
+using System.Collections.ObjectModel;
+
+namespace Kbs.Wpf.User.ViewUser.ViewUserDetail
```
Comment (turtleman31) [inline Kbs.Wpf/User/ViewUser/ReadUserDetail/ReadUserDetailedViewModel.cs:6]: change to read

File: Kbs.Wpf/User/ViewUser/ReadUserIndex/ReadUserIndexPage.xaml [hunk 7]
```diff
@@ -0,0 +1,88 @@
+﻿<Page x:Class="Kbs.Wpf.User.ViewUser.ViewUserGeneral.ViewUserValuesIndexPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+      xmlns:local="clr-namespace:Kbs.Wpf.User.ViewUser.ViewUserGeneral"
+      mc:Ignorable="d" 
+      d:DesignHeight="450" d:DesignWidth="800"
+      Title="ViewUserValuesValuesGeneralViewModel">
```
Comment (turtleman31) [inline Kbs.Wpf/User/ViewUser/ReadUserIndex/ReadUserIndexPage.xaml:9]: change to read

File: Kbs.Wpf/User/ViewUser/ReadUserIndex/ReadUserIndexPage.xaml.cs [hunk 8]
```diff
@@ -0,0 +1,80 @@
+﻿using Kbs.Business.User;
+using Kbs.Wpf.User.ViewUser.ViewUserDetail;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using System.Collections.Generic;
+using Kbs.Data.User;
+
+namespace Kbs.Wpf.User.ViewUser.ViewUserGeneral
```
Comment (turtleman31) [inline Kbs.Wpf/User/ViewUser/ReadUserIndex/ReadUserIndexPage.xaml.cs:9]: change to read

File: Kbs.Wpf/User/ViewUser/ReadUserIndex/ReadUserIndexRoleViewModel.cs [hunk 9]
```diff
@@ -0,0 +1,27 @@
+﻿using Kbs.Business.User;
+
+namespace Kbs.Wpf.User.ViewUser.ViewUserIndex
```
Comment (turtleman31) [inline Kbs.Wpf/User/ViewUser/ReadUserIndex/ReadUserIndexRoleViewModel.cs:3]: change to read

**Review by SilGosker — CHANGES_REQUESTED** [review]

**Review by Katanax28 — APPROVED** [review]

**Pull request merged**